src/coreComponents/mesh/DomainPartition.cpp

```diff
@@ -156,41 +156,43 @@ void DomainPartition::setupCommunications( bool use_nonblocking )
 
   forMeshBodies( [&]( MeshBody & meshBody )
   {
-
-    MeshLevel & meshLevel = meshBody.getBaseDiscretization();
-
-    for( NeighborCommunicator const & neighbor : m_neighbors )
+    if( !meshBody.hasParticles() )
```

Comment:
so, if the `meshBody` has particles, we basically don't need any of the other managers? We clearly need a better way to address this.

Reply:
Yes, this all dates from when I had the impression that a mesh body was a particular discretization scheme. This will certainly change when the `MeshLevel` refactor happens, but I hope it's fine for now.